File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ans = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]: 
                nums[ans] = nums[i] 
                ans += 1
        return ans

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int ans = 1;
        for(int i = 1; i < nums.length; ++i)
            if(nums[i] != nums[i - 1])  {  
                nums[ans] = nums[i]; 
                ++ans; 
            }
        return ans;
    }
}